Jenkinsfile-test

```diff
@@ -1,4 +1,4 @@
-library("tdr-jenkinslib")
+@Library("tdr-jenkinslib@add-before-agent-parameter") _
```

Comment:
Was this a temporary change? Should we remove this commit from the PR?

Reply:
Yeah, I'll take it out now and I'll put in this https://github.com/nationalarchives/tdr-jenkinslib/pull/53/files to fix Jenkinslib as I think the test build will fail without that.